pkg/operators/oci-handler/oci.go

```diff
@@ -120,6 +121,12 @@ func (o *ociHandler) InstanceParams() api.Params {
 			Description: "List of allowed digests, if image digest is not part of it, execution will be denied. By default, all digests are allowed",
 			TypeHint:    api.TypeString,
 		},
+		{
+			Key:         registry,
+			Title:       "Registry",
+			Description: "Gadgets will only be pulled from this specific registry",
```

Comment:
It's not about pulling but about running (if the image was already present on the system it shouldn't be run as well)

```suggestion
			Description: "Only images from this registry will be run",
```